scapy/sendrecv.py

```diff
@@ -94,8 +94,9 @@ def _get_pkt():
         def _get_pkt():
             if bpf_select([pks]):
                 return pks.recv()
-    elif conf.use_pcap or (not isinstance(pks, StreamSocket)
-                           and (DARWIN or FREEBSD or OPENBSD)):
+    elif (conf.use_pcap and not isinstance(pks, StreamSocket) and
```

Comment:
`not isinstance(pks, StreamSocket) and not isinstance(pks, L3RawSocket)` can be written as `not isinstance(pks, (StreamSocket, L3RawSocket))`.